Python/crossfire/crossfire/occurrences.py

```diff
@@ -0,0 +1,32 @@
+from queue import Queue, Empty
+
+
+class Occurrences:
+    def __init__(self, client):
+        self.client = client
+        self.buffer = Queue()
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        if self.buffer.empty():
+            self.load_occurrences()
+
+        try:
+            occurrences = self.buffer.get_nowait()
+        except Empty:
+            raise StopIteration
+
+        return occurrences
+
+    def load_occurrences(self):
+        occurrences = self.client.get(
+            "https://api-service.fogocruzado.org.br/api/v2/occurrences?"
```

Comment:
Usar a constante, remover caracter não utilizado

```suggestion
            f"{client.URL}/occurrences"
```